js/search/search.module.js

```diff
@@ -38,6 +38,9 @@
                 scrollTop:   'last',
                 persistent:  true,
                 history:     false,
+                params: {
+                    searchInCaptions: null
```

Comment:
`searchInCaptions` VS `searchWithCaptions`

Reply:
default to false instead of null

Reply:
Agree with both

Reply:
@ChristiaanScheermeijer `includeCaptionMatches`?